### DISCUSSION TITLE
Username to not show

### DISCUSSION THREAD

**Discussion by mskenderian-wps** [linuxserver/Heimdall#719] [Ideas]
Anyway we can configure it, so the username does not show on the frontpage. so users have to type in a username and password like every other app.

**Comment by sebastianschauenburg** [comment]
Would be a nice addition indeed. Meanwhile you could deny access to the /userselect Location (in Apache), which at least prevents the userlist from showing.

**Comment by Ataraxiall** [comment]
Hi,

Im interested in this funtion too.

Im running docker, but i dont know how to denny access to /userselect.can helpme?

Thanks!
